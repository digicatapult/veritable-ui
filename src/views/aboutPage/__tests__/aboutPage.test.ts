import { expect } from 'chai'
import { describe, it } from 'mocha'
import AboutPageTemplates from '../about.js'

describe('AboutPageTemplates', () => {
  it('should render about page', async () => {
    const templates = new AboutPageTemplates()
    const rendered = await templates.aboutPage()
    expect(rendered).to.matchSnapshot()
  })
})
