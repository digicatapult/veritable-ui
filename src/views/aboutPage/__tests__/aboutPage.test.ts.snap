// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPageTemplates should render about page 1`] = `"<!DOCTYPE html><html lang=\\"en\\"><head><script src=\\"/lib/htmx.org/htmx.min.js\\"></script><script src=\\"/lib/htmx-ext-json-enc/json-enc.js\\"></script><script src=\\"/public/scripts/auth-redirect.js\\"></script><link rel=\\"icon\\" type=\\"image/ico\\" href=\\"/public/images/favicon-dark.ico\\" media=\\"(prefers-color-scheme: dark)\\"/><link rel=\\"icon\\" type=\\"image/ico\\" href=\\"/public/images/favicon.ico\\" media=\\"(prefers-color-scheme: light), (prefers-color-scheme: no-preference)\\"/><link rel=\\"stylesheet\\" type=\\"text/css\\" href=\\"/public/styles/main.css\\"/><link rel=\\"stylesheet\\" type=\\"text/css\\" href=\\"/public/styles/about.css\\"/><title>Veritable - About</title></head><body hx-ext=\\"json-enc\\"><nav id=\\"side-bar\\"><a id=\\"veritable-logo\\" href=\\"/about\\"></a><div class=\\"profile-icon\\" style=\\"background-color:#38b6ff;\\">A</div><a title=\\"categories\\" href=\\"/\\" data-active style=\\"--background-image:url('/public/images/category.svg');\\"></a><a title=\\"connections\\" href=\\"/connection\\" style=\\"--background-image:url('/public/images/connection.svg');\\"></a><a title=\\"queries\\" href=\\"/queries\\" style=\\"--background-image:url('/public/images/query.svg');\\"></a><a title=\\"credentials\\" href=\\"/credentials\\" style=\\"--background-image:url('/public/images/credential.svg');\\"></a><a title=\\"settings\\" href=\\"/settings\\" style=\\"--background-image:url('/public/images/setting.svg');\\"></a></nav><main><div id=\\"content-header\\"><h1>About Veritable</h1><div id=\\"content-header-nav\\"><a title=\\"home\\" class=\\"nav icon\\" href=\\"/\\"></a><a title=\\"\\" href=\\"/\\"></a></div></div><div id=\\"content-main\\"><div id=\\"about-container\\"><img src=\\"/public/images/logo-square.svg\\"/><h3>About veritable </h3><p>Veritable is a platform that enhances trust and transparency in supply chains through secure, privacy-preserving data sharing and process automation. With verified digital identities, automated queries, and credential management, Veritable enables verified participants to collaborate with confidence, safeguarding sensitive information and ensuring compliance.</p><div id=\\"key-features-container\\"><h3>Key Features</h3><div class=\\"in-row\\"><h3>Verified Onboarding: </h3><p> Verifies all participants, building trust across the supply chain.</p></div><div class=\\"in-row\\"><h3>Automated Queries:</h3><p>Enables seamless data requests and responses.</p></div><div class=\\"in-row\\"><h3>Credential Management: </h3><p>Manages and verifies digital credentials for compliance.</p></div><div class=\\"in-row\\"><h3>Self-Managed Identities: </h3><p>Empowers businesses to control their identities and data, supporting secure, decentralised collaboration.</p></div></div><i>Veritable's core principles ensure secure, user-friendly supply chain collaboration. With privacy-preserving data sharing, robust security, and transparent verifiability, Veritable protects sensitive information while guaranteeing authenticity and trust among all participants.</i></div></div></main></body></html>"`;
